specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+        },
+        "id": {
+          "type": "string"
+        },
+        "longRunningOperationType": {
+          "description": "Represents a category of long running operation",
+          "enum": [
+            "Purchase",
+            "ReleasePhoneNumber",
+            "Search",
+            "UpdatePhoneNumberCapabilities"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "LongRunningOperationType",
+            "modelAsString": true
+          }
+        },
+        "lastActionDateTime": {
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SearchRequest": {
+      "description": "Represents a phone number search request.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumberType": {
+          "description": "The phone number type of the search.",
+          "enum": [
+            "Geographic",
+            "TollFree",
+            "Unknown"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
+          "description": "The assignment type of the search.",
+          "enum": [
+            "Unknown",
+            "User",
+            "Application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "areaCode": {
+          "description": "The area code of the search.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of phone numbers in the search.",
+          "maximum": 2147483647,
+          "minimum": 1,
+          "type": "integer"
+        }
+      }
+    },
+    "SearchResponse": {
+      "description": "Represents a search response.",
+      "type": "object",
+      "properties": {
+        "searchId": {
+          "description": "The search id of the search that was created.",
+          "type": "string"
+        },
+        "phoneNumbers": {
+          "description": "The available phone numbers retrieved from the search.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "phoneNumberType": {
+          "description": "The phone number type of the search.",
+          "enum": [
+            "Geographic",
+            "TollFree",
+            "Unknown"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
```

Comment:
use ref

Reply:
As discussed offline, will leave as-is since Swashbuckle can't generate common parameter easily.